mllib/src/main/scala/org/apache/spark/mllib/tree/loss/LogLoss.scala

```diff
@@ -20,6 +20,12 @@ package org.apache.spark.mllib.tree.loss
 import org.apache.spark.annotation.{DeveloperApi, Since}
 import org.apache.spark.mllib.util.MLUtils
 
+/**
+ * Trait for adding probability function for the gradient boosting algorithm.
```

Comment:
nit: Doc is too specific. It's a trait for classification loss, not necessarily just for adding a prob func. I'm not sure we need a doc here. Actually we could add a doc to the `computeProbability` method saying that it computes the class probability given the margin, if that's unclear.